pkgs/os-specific/linux/zsa-udev-rules/default.nix

```
@@ -15,6 +15,9 @@ stdenv.mkDerivation rec {
   dontConfigure = true;
   dontBuild = true;
   dontFixup = true;
+  patches = [
+    ./voyager.patch
```

Sorry for not being clear on my idea, i meant something like this

```suggestion
    # https://github.com/zsa/wally/pull/135
    (fetchpatch {
      url =
        "https://github.com/zsa/wally/commit/bceec979b283123da45502e236cf24afad753f2b.patch";
      sha256 = "sha256-ZkzKffTobhJEKPsvLwZT0QzlvCU5BOh0/9HBFQvH+KM=";
    })
```

As we have already a downloadable patch from Github, we can and should use that instead of duplicating the code here.
Instead of the commit we could also use the PR changes, but that would break if upstream requests changes.

---

TIL!